web_external/Datasets/UploadDatasetMetadataView.js

```diff
@@ -0,0 +1,195 @@
+isic.views.UploadDatasetMetadataView = isic.View.extend({
```

Comment:
Let's rename this View, and all associated JS, CSS, and Jade files (and probably some HTML classes and IDs :slightly_frowning_face: ) to be something like `RegisterMetadataDatasetView`, as:
* we're referring to this process internally on the server-side as "registering" metadata, rather than just "uploading" it
* the ordering of words makes the place of this functionality within the logical site hierarchy more clear